js/game.js

```diff
@@ -0,0 +1,256 @@
+/*global window, console, document, Image
+*/
+
+// Game object constructor
+// "Revealing module pattern"
+function createGame () {
```

Comment:
You can do it IAN!! 

![image](https://cloud.githubusercontent.com/assets/970858/10504804/f903419e-7337-11e5-8611-ddc8528beecd.png)

:bomb: